google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugProcessHandler.java

```diff
@@ -55,10 +57,15 @@ public CloudDebugProcessHandler(CloudDebugProcess process) {
         LOG.error("userEmail is null. To launch a debug session user needs to be logged in");
       }
     } else {
-      LOG.error("process or process state is null. This should only happen in tests");
+      LOG.error("process state is null.");
     }
   }
 
+  @VisibleForTesting
+  CloudDebugProcessHandler() {
+    this.process = null;
```

Comment:
sure, it was because the field is final, but I can remove that if you think its less ugly